### PR TITLE
Clean up component props

### PR DESCRIPTION
**Pull request by victoria-dos**
https://issues.redhat.com/browse/RHCLOUD-19008

### CODE REVIEW

File: src/smart-components/role/add-role/add-role-wizard.js [hunk 1]
```diff
@@ -14,13 +14,13 @@ import { WarningModal } from '../../common/warningModal';
 import '../../common/hideWizard.scss';
 
 const AddRoleWizard = ({ addNotification, createRole, history: { push }, pagination }) => {
-  const [formData, setValues] = useState({});
+  const [formData, setFormData] = useState({});
```
[the remark this answers is not in the document]
Comment (victoria-dos) [reply src/smart-components/role/add-role/add-role-wizard.js:17]: just thought it is a convention to name it like this, isn't it?

[the remark this answers is not in the document]
Comment (karelhala) [reply src/smart-components/role/add-role/add-role-wizard.js:17]: Yeah, didn't notice the first name.

File: src/presentational-components/shared/toolbar.js [hunk 2]
```diff
@@ -305,16 +305,7 @@ Toolbar.propTypes = {
...
-    PropTypes.shape({
-      id: PropTypes.string,
-      label: PropTypes.element,
-      value: PropTypes.string,
-      type: PropTypes.string.isRequired,
-      filterValues: PropTypes.object,
-      placeholder: PropTypes.string,
-    })
-  ),
+  filterItems: PropTypes.arrayOf(PropTypes.object),
```
[the remark this answers is not in the document]
Comment (Hyperkid123) [reply src/presentational-components/shared/toolbar.js:308]: @karelhala I would agree, but there is no point. I talked to @aneelac22 and want to kick off TS migration for RBAC soon-ish.

[the remark this answers is not in the document]
Comment (karelhala) [reply src/presentational-components/shared/toolbar.js:308]: If we're going to migrate to TS in near future I am fine with adding just `object` in here.

[the remark this answers is not in the document]
Comment (Hyperkid123) [reply src/presentational-components/shared/toolbar.js:308]: I promise we will. Soon

**Review by karelhala — APPROVED** [review]

**Pull request merged**